### PR TITLE
feat(angular): add mfe migration to fix incorrect setups

### PR DESCRIPTION
**Pull request by Coly010**
## Current Behavior

Missing a migration to fix existing incorrect mfe setup.

## Expected Behavior

Add the needed migration.

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+    return webpackConfig;
+  }
+
+  webpackConfig = `${webpackConfig.slice(
+    0,
+    optimizationObjectNode.getStart() + 1
+  )}\nminimize: true,${webpackConfig.slice(
+    optimizationObjectNode.getStart() + 1
+  )}`;
```
Comment (leosvelperez) [inline packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts:220]: I guess we don't need to do this? It's the default value and we don't generate it for new projects.

File: packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  )})${webpackConfig.slice(sharedObjectNode.getEnd())}`;
+
+  // wrap the shared libs in the function
+  ast = tsquery.ast(webpackConfig);
+  const requirePathStatement = tsquery(ast, REQUIRE_PATH_AST_QUERY, {
+    visitAllChildren: true,
+  })[0];
+  webpackConfig = `${webpackConfig.slice(
+    0,
+    requirePathStatement.getEnd()
```
Comment (leosvelperez) [inline packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts:273]: I wouldn't rely on the `path` import to add this. The configuration imports might deviate from what was initially generated, the imports sorted in a different way, new imports added, etc.

I think we could add this chunk of code right before the `module.exports` or at the very least, after the `@angular-architects/module-federation/webpack` import.

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+  const projects: Record<string, string> = {};
+  forEachExecutorOptions(
+    tree,
+    NRWL_WEBPACK_BROWSER_BUILDER,
+    (opts, projectName) => {
+      // Update the webpack config
+      const webpackPath = opts[CUSTOM_WEBPACK_OPTION]?.path;
+      if (!webpackPath || !tree.exists(webpackPath)) {
+        logger.warn(
+          `Webpack config file for project: ${projectName} does not exist. Skipping project.`
```
[the remark this answers is not in the document]
Comment (Coly010) [reply packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts:25]: Sounds good to me

File: packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts (hunk 2 above)
Comment (Coly010) [reply packages/angular/src/migrations/update-13-5-0/update-mfe-configs.ts:273]: That might be safer, agreed.

**Review by leosvelperez — APPROVED** [review]
LGTM! 🎉

**Pull request merged**